docs: Custom feedback (#23169)
bigbluebutton/bigbluebutton: docs/docs/new-features.md

#### Feedback form removed

We have removed the feedback form that used to be part of the client. It was relying on client logs to carry the information and was not particularly flexible. See https://github.com/bigbluebutton/bigbluebutton/pull/22111 for more information.
A new repository was contributed by Mconf https://github.com/bigbluebutton/custom-feedback with a much more sophisticated feedback form (see below).

#### Custom feedback

> In BigBlueButton 3.0 a new way of collecting feedback from users was introduced. It's a standalone, customizable, extensible application that can be integrated into BigBluebutton. Please refer to its [README](https://github.com/bigbluebutton/custom-feedback/blob/master/README.md) for details on how to customize and install it.

Reviewer: ```suggestion
In BigBlueButton 3.0 we replaced the old feedback form with a new way of collecting feedback from users. It's a standalone, customizable, extensible application that can be integrated into BigBlueButton. Please refer to its [README](https://github.com/bigbluebutton/custom-feedback/blob/master/README.md) for details on how to customize and install it.
```
